DXRModules/DeusEx/Classes/DXRFlags.uc

```diff
@@ -563,7 +561,7 @@ function FlagsSettings SetDifficulty(int new_difficulty)
         bingo_scale = 0;
 
         SetGlobalSeed("random starting map");
-        settings.starting_map = class'DXRStartMap'.static.ChooseRandomStartMap(dxr);
+        settings.starting_map = class'DXRStartMap'.static.ChooseRandomStartMap(dxr,10);
```

Comment:
why doesn't Walton Ware allow Liberty Island? I guess it can do it on NG+ anyways?

Reply:
It's mostly just because of the Advanced selection screen.  If we allowed Liberty Island, we'd need a "vanilla" Liberty Island (0) and "starting map" Liberty Island (10) in the selections, or somehow fake around it.  Mostly just for simplicity.  Maybe makes it a bit more clear that the "random starting map" worked too.